static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -115,9 +115,9 @@ define([
         };
 
         if (overlay) {
-            var formattedDuration = getFormattedDuration(players[atomId].player.getDuration());
-            
-            setDuration(formattedDuration, overlay);
+            if (!config.page.isFront) {
```

Comment:
We could work this out server side and pass it down as a parameter on the template.